### ISSUE TITLE
Make the TOC more noticeable

### ISSUE THREAD

**Issue by phillipskevin** [canjs/bit-docs-html-canjs#576]
During user testing, users did not notice the "On This Page" sidebar. I think we should make it slightly larger or make the highlighting more animated.

I'm open to other ways of accomplishing this.

**Comment by justinbmeyer** [comment]
Was it important for them to see? 

Making it more visible might distract from the main content.  There's a tradeoff here that might not be worth it.  I think it's important to identify what went wrong.

**Comment by phillipskevin** [comment]
Yes, that's a good point that I should have mentioned. One example is a user was on the [API Docs page](http://next.canjs.com/doc/api.html) and was trying to figure out how to load some data from an API and ended up searching and finding `can-ajax` in the Utilities section and never found the Data Modeling header in the TOC.
